packages/superset-ui-core/src/query/normalizeOrderBy.ts

```diff
@@ -53,6 +54,19 @@ export default function normalizeOrderBy(queryObject: QueryObject): QueryObject
     };
   }
 
+  // todo: Removed `legacy_ordery_by` after refactoring
```

Comment:
﻿This part of the logic will be refactored in other [PR](https://github.com/apache-superset/superset-ui/pull/1338) .